libsolidity/formal/CHC.cpp

```diff
@@ -749,7 +750,12 @@ void CHC::externalFunctionCall(FunctionCall const& _funCall)
 		return;
 	}
 
-	FunctionType const& funType = dynamic_cast<FunctionType const&>(*_funCall.expression().annotation().type);
+	FunctionCallOptions const* callOptions = nullptr;
+	Expression const* callExpr = &_funCall.expression();
+	if ((callOptions = dynamic_cast<FunctionCallOptions const*>(callExpr)))
```

Comment:
```suggestion
	if (callOptions = dynamic_cast<FunctionCallOptions const*>(callExpr))
```

Reply:
The compiler complains if I don't put the extra pair.

Reply:
It probably just tells you that using `=` inside `if` is evil :P

Reply:
It actually asked if I meant `==`

Reply:
That's the same thing :)